### PR TITLE
add new internal issue template and disable blank issues

### PR DESCRIPTION
**Pull request by zkat**
Template preview: https://github.com/NuGet/Home/blob/zkat/internal-template/.github/ISSUE_TEMPLATE/INTERNAL.yml

### CODE REVIEW

**Review by nkolev92 — COMMENTED** [review]
Any way I can see a sample issue created through this template?

I'm not sure what problem we're solving here. 

We have a lot of duplication about labeling etc. 
I would also prefer *not* to have to do a click through to create an issue. I think this is an overkill right now and I think we should keep the blank issues.

**Comment by zkat** [comment]
None of the template text will show up in the issue. This is basically equivalent to a standard issue, but with a block of reminder text preceding it.

Note that if you're clicking on `new issue` to create an issue, you still have to do the same number of click throughs: one.

File: .github/ISSUE_TEMPLATE/INTERNAL.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      value: |
+        ## Preflight Checklist
+
+        **Do not file this issue until you've gone through this checklist.**
+
+        ### Basic Labels (required)
+
+        * Add **Regression** labels if this is a regression.
+        * Add a **Type:** label
+        * Add **Area:** and/or **Functionality:** labels
```
Comment (nkolev92) [inline .github/ISSUE_TEMPLATE/INTERNAL.yml:15]: Product probably deserves a mention here. 

Area is usually less important than Functionality and Product.

File: .github/ISSUE_TEMPLATE/INTERNAL.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    attributes:
+      value: |
+        ## Preflight Checklist
+
+        **Do not file this issue until you've gone through this checklist.**
+
+        ### Basic Labels (required)
+
+        * Add **Regression** labels if this is a regression.
+        * Add a **Type:** label
```
Comment (nkolev92) [inline .github/ISSUE_TEMPLATE/INTERNAL.yml:14]: I think right now, it's either `Type` or `Category:Engineering`. 

I'd prefer to have had Type:Engineering, but I can't make that call :)

**Review by nkolev92 — COMMENTED** [review]
Minor feedback on the template. 

My primary concern is velocity, so as long as that's not affected, I'm fine with whatever the team finds comfortable.

File: .github/ISSUE_TEMPLATE/INTERNAL.yml (hunk 2 above)
Comment (nkolev92) [reply .github/ISSUE_TEMPLATE/INTERNAL.yml:14]: We're having the `Type:Engineering` discussion in https://teams.microsoft.com/l/message/19:00d019d741b44e54baadd67ef7aeb41d@thread.skype/1634250158470?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ad34ac55-6785-4e7c-82c3-7ba405d30c3c&parentMessageId=1634250158470&teamName=NuGet&channelName=Client&createdTime=1634250158470

**Comment by zivkan** [comment]
Closing as stale. If someone else feels passionate about this, please re-open or create a new PR.

**Pull request closed without merging**